docs/content/preview/yugabyte-voyager/reference/bulk-data-load/import-data-file.md

```diff
@@ -0,0 +1,292 @@
+| --target-db-name <name> | Target database name. |
+| --target-db-password <password>| Target database password. Alternatively, you can also specify the password by setting the environment variable `TARGET_DB_PASSWORD`. If you don't provide a password via the CLI during any migration phase, yb-voyager will prompt you at runtime for a password. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes. |
+| --target-db-port <port> | Port number of the target database machine. (default: 5433) |
+| --target-db-schema <schemaName> | Schema name of the target database. MySQL and Oracle migrations only. |
+| --target-db-user <username> | Username of the target database. |
+| [--target-ssl-cert](../../yb-voyager-cli/#ssl-connectivity) <certificateName> | Name of the certificate which is part of the SSL `<cert,key>` pair. |
+| [--target-ssl-key](../../yb-voyager-cli/#ssl-connectivity) <keyName> | Name of the key which is part of the SSL `<cert,key>` pair. |
+| [--target-ssl-crl](../../yb-voyager-cli/#ssl-connectivity) <path> | Path to a file containing the SSL certificate revocation list (CRL).|
+| [--target-ssl-mode](../../yb-voyager-cli/#ssl-connectivity) <SSLmode> | One of `disable`, `allow`, `prefer`(default), `require`, `verify-ca`, or `verify-full`. |
+| [--target-ssl-root-cert](../../yb-voyager-cli/#ssl-connectivity) <path> | Path to a file containing SSL certificate authority (CA) certificate(s). |
+| --verbose | Display extra information in the output. (default: false) |
+| -y, --yes  | Answer yes to all prompts during the export schema operation. |
+
+## Examples
+
+The following examples use the `--data-dir` argument along with `--file-table-map` argument and contextually differ based on whether the data is imported from local disk or cloud storage options.
+
+{{< tabpane text=true >}}
+
+ {{% tab header="CSV" %}}
+
+Import data for CSV files using `import data file` command by providing the argument `--format 'csv'` in the command.
+
+### Import data file from local disk
+
+The `--data-dir` argument is a path to the local directory where all the CSV files are present, and the `--file-table-map` argument provides a comma-separated mapping between each CSV file in `--data-dir` to the corresponding table in the database (you can mention case-sensitive table names as well, for example: `--file-table-map 'foo.csv:"Foo"'`); each file has a header, delimiter as `','` (default), escape and quote character as `'"'`(default).
+
+```sh
+yb-voyager import data file --export-dir '/dir/export-dir' \
+        --target-db-host 'x.x.x.x' \
```

Comment:
pls change all the occurrences of these in the examples - 
1. `--target-db-host 'x.x.x.x'` to `--target-db-host '127.0.0.1'`,
2. `--export-dir '/dir/export-dir'` to `--export-dir /dir/export-dir`
3.  `--data-dir '/dir/data-dir'` to `--data-dir /dir/data-dir`